react/src/Table/utils/omitInteractivePseudos.ts

```diff
@@ -0,0 +1,54 @@
+import { DOMAttributes } from 'react'
+import { SystemStyleObject } from '@chakra-ui/react'
+import { omit } from 'lodash'
+
+type InteractableProps = Pick<
+  DOMAttributes<unknown>,
+  | 'onClick'
+  | 'onKeyDown'
+  | 'onKeyUp'
+  | 'onKeyPress'
+  | 'onMouseDown'
+  | 'onMouseUp'
+  | 'onMouseEnter'
+  | 'onMouseLeave'
+  | 'onMouseMove'
+  | 'onMouseOver'
+  | 'onMouseOut'
+  | 'onFocus'
+  | 'onBlur'
+>
+
+const isInteractive = (props: InteractableProps) => {
+  return (
+    !!props.onClick ||
+    !!props.onKeyDown ||
+    !!props.onKeyUp ||
+    !!props.onKeyPress ||
+    !!props.onMouseDown ||
+    !!props.onMouseUp ||
+    !!props.onMouseEnter ||
+    !!props.onMouseLeave ||
+    !!props.onMouseMove ||
+    !!props.onMouseOver ||
+    !!props.onMouseOut ||
+    !!props.onFocus ||
+    !!props.onBlur
+  )
+}
+
+const INTERACTIVE_PSEUDO_PROPS = ['_active', '_hover', '_pressed', '_selected']
```

Comment:
is `_focus` not included here? from mdn [docs](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes#user_action_pseudo-classes), it seems to be interactable (also, should we base this off the mdn one?)